exonum/src/helpers/fabric/details.rs

```diff
@@ -514,6 +568,16 @@ impl Command for GenerateNodeConfig {
             .arg::<String>(SERVICE_KEY_PATH)
             .unwrap_or_else(|_| "service.toml".into())
             .into();
+        let consensus_key_pass_method: PassInputMethod = context
+            .arg::<String>(CONSENSUS_KEY_PASS_METHOD)
+            .unwrap_or_else(|_| "".to_owned())
```

Comment:
Why not just `String::new`?

Reply:
Or `.unwrap_or_default()` could be ok as well here.